server/chat-commands/core.ts

```diff
@@ -1265,12 +1263,10 @@ export const commands: ChatCommands = {
 		if (!user.named) {
 			return this.popupReply(`You must choose a username before you challenge someone.`);
 		}
-		if (Config.pmmodchat) {
-			if (Users.globalAuth.atLeast(user, Config.pmmodchat as GroupSymbol)) {
-				const groupName = Config.groups[Config.pmmodchat].name || Config.pmmodchat;
-				this.popupReply(`Because moderated chat is set, you must be of rank ${groupName} or higher to challenge users.`);
-				return false;
-			}
+		if (Config.pmmodchat && !user.hasSysopAccess() && !Users.globalAuth.atLeast(user, Config.pmmodchat as GroupSymbol)) {
+			const groupName = Config.groups[Config.pmmodchat].name || Config.pmmodchat;
+			this.popupReply(`Because moderated chat is set, you must be of rank ${groupName} or higher to challenge users.`);
```

Comment:
"Moderated chat" is a term from IRC and never actually made sense to me; we might want to think of a better word for it in general.

Reply:
"Authfilter"?

Reply:
Considering the command is called `/modchat`, I think just calling it modchat (as in only **mod**s [or anyone with rank > X] can **chat**) would make sense. I think having a `/modchat` command to enable authfilter would just be confusing, although we could rename the command, I suppose. (Also, the namechange should probably go in a different commit.)

Reply:
Yeah, I know. I think for now we can stick with `modchat`, but I've been leaning towards not calling it that in error messages, as you've seen here.